File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.js
/**
 * @param {number[]} numbers
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(numbers, target) {
   let l = 0
   let r = numbers.length-1
   while (r > l){
       if (numbers[r] + numbers[l] === target) return [l+1,r+1]
       if (numbers[r] + numbers[l] > target){
           r--
       }else{
           l++
       }
   }
    
};

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
        dict = {}
        for i in range(0, len(numbers)):
            if target-numbers[i] in dict:
                return [dict.get(target-numbers[i])+1, i+1]
            dict[numbers[i]] = i
        